pkg/config/parameters.go

```diff
@@ -525,7 +525,10 @@ type NetworkParameters struct {
 	//
 	// See https://www.envoyproxy.io/docs/envoy/v1.17.0/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto?highlight=xff_num_trusted_hops
 	// for more information.
-	XffNumTrustedHops uint32 `yaml:"num-trusted-hops"`
+	XffNumTrustedHops uint32 `yaml:"num-trusted-hops,omitempty"`
+
+	// Configure the port used to access the Envoy Admin interface.
```

Comment:
This will default to "on" if you don't specify. The only way to turn it off is to make the port  "0". This matches what we do today, but offers up a way for folks to be "more secure" if they'd like. 

I'm working on explaining those bits in the the docs PR which is turning into a bunch of work. =)